api/pkg/s3/objectversionget.go

```diff
@@ -0,0 +1,153 @@
+// Copyright 2019 The OpenSDS Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package s3
+
+import (
+	"io"
+	"net/http"
+	"strings"
+	"time"
+
+	"github.com/emicklei/go-restful"
+	"github.com/opensds/multi-cloud/api/pkg/common"
+	. "github.com/opensds/multi-cloud/api/pkg/s3/datatype"
+	s3error "github.com/opensds/multi-cloud/s3/error"
+	pb "github.com/opensds/multi-cloud/s3/proto"
+	log "github.com/sirupsen/logrus"
+)
+
+// GetObjectVersionHandler - GET Object
+func (s *APIService) ObjectVersionGet(request *restful.Request, response *restful.Response) {
```

Comment:
Can we made a little modification to ObjectGet, instead of creating a totaly new function here?